public/app/features/dashboard-scene/serialization/layoutSerializers/RowsLayoutSerializer.test.ts

```diff
@@ -39,7 +42,14 @@ describe('deserialization', () => {
             spec: {
               title: 'Row 1',
               collapsed: false,
-              layout: { kind: 'ResponsiveGridLayout', spec: { row: '', col: '', items: [] } },
+              layout: {
+                kind: 'ResponsiveGridLayout',
+                spec: {
+                  row: 'minmax(min-content, max-content)',
+                  col: 'repeat(auto-fit, minmax(400px, 1fr))',
```

Comment:
oh lord, will i have to know css to use this layout as-code?! 😿

Reply:
I hope not. I think we are going to rework how row/cols work in the responsive grid.